packages/office-ui-fabric-react/src/components/pickers/TagPicker/TagItem.scss

```diff
@@ -2,15 +2,18 @@
 
 .root {
   @include focus-border();
-  flex-shrink: 0;
+  flex-shrink: 1;
   background: $ms-color-neutralLighter;
   height: 32px;
   line-height: 32px;
   cursor: default;
   user-select: none;
   display: flex;
   flex-wrap: nowrap;
+  min-width: 100px;
   max-width: 100%;
+  margin: 2px 2px;
```

Comment:
Also you're adding margin on the root of the tag item. I think this means an inner padding of 4px, but outer padding of 2px? I think this might have visual regression implications so please double check here that you haven't accidentally changed something. Maybe a screenshot of before/after would be helpful.